### DISCUSSION TITLE
Issue with Extracting Light-Kappa Chains in Dog

### DISCUSSION THREAD

**Discussion by MikolajKocikowski** [milaboratory/mixcr#1613] [General]
Hi,

I've been using the imgt.202312-3.sv8.json germline database with the latest MiXCR (Oxford Nanopore preset) for extracting heavy and light chains from dog samples. While I've successfully extracted heavy and light-lambda chains, I encountered issues with light-kappa chains. Specifically, of the 10 samples, only one produced a light-kappa file, containing a minimal number of sequences (one or two), which seems like an artifact. Given that some of the sequenced samples are known to contain kappa chains, this result is unexpected.

I'm looking for the culprit behind the lack of kappa chains. Any advice would be greatly appreciated, thanks!

**Comment by mizraelson** [comment]
Hi, can you share the align report for the sample where you expect the missing chain?

**Comment by MikolajKocikowski** [comment]
Hi! I attach two files from samples known to contain kappas among other chains. Barcode09 file produced the IGK output file with one chain, Barcode10 did not. Moderate alignment success rate is expected due to the nature of the library. Samples contain scFvs and some single chain constructs. Thanks for taking a look!

[barcode09_result.align.report.txt](https://github.com/milaboratory/mixcr/files/14760099/barcode09_result.align.report.txt)
[barcode10_result.align.report.txt](https://github.com/milaboratory/mixcr/files/14760101/barcode10_result.align.report.txt)

**Comment by mizraelson** [comment]
I see, well the successfully aligned reads rate is rather low indeed here. Is it possible to identify a read that covers IGK from the the original FASTQ file but appears not to be aligned correctly?

**Reply by MikolajKocikowski** [reply]
Hi Mark, I apologize for the delay - the issue is still relevant, and I'll be able to answer early next weeks. Many thanks for your time!
